### PR TITLE
Fix the issue when mount an empty oss bucket

### PR DESCRIPTION
**Pull request by Jackson-Wang-7**
### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: dora/underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java [hunk 1]
```diff
@@ -395,6 +395,10 @@ public Boolean hasNextChunk() {
   @Override
   protected ObjectStatus getObjectStatus(String key) {
     try {
+      if (isRoot(key)) {
+        // return a virtual root object
+        return new ObjectStatus(key, null, 0, null);
```
Comment (yyongycy) [inline dora/underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java:400]: what are the attributes from the user side?

Comment (Jackson-Wang-7) [reply dora/underfs/oss/src/main/java/alluxio/underfs/oss/OSSUnderFileSystem.java:400]: user can not see the root directly attribute

**Review by yyongycy — APPROVED** [review]
lgtm

**Comment by Jackson-Wang-7** [comment]
alluxio-bot, merge this please

**Pull request merged**